src/util.rs

```diff
@@ -274,6 +275,42 @@ impl FromStr for GitVersion {
     }
 }
 
+/// Returns a path for a given file, searching through PATH to find it.
+pub fn get_from_path(exe_name: &str) -> Option<PathBuf> {
+    env::var_os("PATH").and_then(|paths| {
+        env::split_paths(&paths).find_map(|dir| {
+            let bash_path = dir.join(exe_name);
+            if bash_path.is_file() {
+                Some(bash_path)
+            } else {
+                None
+            }
+        })
+    })
+}
+
+/// Returns the path to bash.
+pub fn get_bash() -> Option<PathBuf> {
+    let exe_name = if cfg!(target_os = "windows") {
+        "bash.exe"
+    } else {
+        "bash"
```

Comment:
Not all Unix systems will have `bash` installed (some will have `dash` instead). But they should have `sh`, in order to be POSIX-compatible. Let's change this function to return `sh` in this case.

Reply:
Switch to sh for the !windows case.